### ISSUE TITLE
RC Test Ohms Law 1.4.0-rc.3

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#327]
@emily-phet @terracoda @jbphet @KatieWoe Ohms Law 1.4.0-rc.3 is ready for an RC spot check. This comes after #313. rc.2 was skipped only because I encountered an error during the build process while trying to build RC 2.

Changes from the last RC are small enough that we don't need to do another full RC. Please just verify that the following issues are resolved and then a quick spot check that all seems well.

Link to RC: https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.3/phet/ohms-law_en_phet.html

Issues to spot check:
- [x] https://github.com/phetsims/ohms-law/issues/142 (sound with mobile VoiceOver)
- [x] https://github.com/phetsims/ohms-law/issues/140 (?stringTest overlap)
- [x] https://github.com/phetsims/ohms-law/issues/54 (Credits Update)

Assigning to @ariel-phet to prioritize.

**Comment by KatieWoe** [comment]
- [ ] Win 10 chrome (with and without Jaws)
- [ ] Win 10 firefox (with and without NVDA)
- [ ] Win 10 edge
---
- [ ] Mac 10.14 chrome
- [ ] Mac 10.14 firefox
- [ ] Mac 10.14 safari (with and without VO)
---
- [ ] Win 7 IE
- [ ] Mac 10.11 safari
---
- [ ] ChromeOS
- [ ] iOS 12 (with and without VO)
---
@jessegreenberg to approve test devices

**Comment by jessegreenberg** [comment]
A quick check over the above list would be great @KatieWoe, thanks.

**Comment by KatieWoe** [comment]
Some odd behavior from jaws.  Highlights large and seemingly unrelated area.  Also https://github.com/phetsims/resistance-in-a-wire/issues/177 shows up, but the actual value isn't read (if I recall correctly, it should).  @jessegreenberg to determine if an issue should be made.
https://drive.google.com/file/d/116L6UpV235rqAKw6ILUWwsoO_X7LCbL4/view?usp=sharing

**Comment by KatieWoe** [comment]
@jessegreenberg should iframe and xhtml be tested as well?

**Comment by jessegreenberg** [comment]
Thanks  @KatieWoe , I will take a look at the above issue. No need to test xhtml and iframes, changes were not substantial enough.

**Comment by KatieWoe** [comment]
QA is done

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, Ill make an issue for https://github.com/phetsims/QA/issues/327#issuecomment-496708690, then close this.

**Comment by jessegreenberg** [comment]
Issues made, closing.
